src/math/abs.jl

```diff
@@ -32,11 +32,14 @@ abs_max([-3,1,2]) # returns -3
 abs_max([-7,-3,6]) #returns -7
 ```
 
-Contributed By:- [Ashwani Rathee](https://github.com/ashwani-rathee)
+Contributed By:- [Ashwani Rathee](https://github.com/ashwani-rathee) and [Rratic](https://github.com/Rratic)
 """
-function abs_max(x)
-    result = sort(x, by=abs; alg=QuickSort)
-    return result[length(result)]
+function abs_max(x::Vector{N})where N<:Number
```

Comment:
I guess Number can be changed to Real becuase comparing Complex does not make sense

Reply:
Unless there is a conflict with another `abs_max` method, there is no need to provide types